public/javascripts/global.js

```diff
@@ -172,14 +172,14 @@ function fetchQuiz(event) {
     if (target.hasClass('quiz-btn')){
       var team = $('.selected-team'),
           id = team.data('team-id'),
-          league = team.data('league');
-
+          league = team.data('league'),
+          rb_team_id = team.data('id');
```

Comment:
is this right? We pull rb_team_id from the id data attribute? Above we pull id from the team-id data attribute? I'm confused.